tests/functional/adapter/streaming_tables/test_st_changes.py

```diff
@@ -33,6 +32,7 @@ def check_start_state(project, streaming_table):
         _check_tblproperties(results.config["tblproperties"], {"key": "value"})
         assert results.config["refresh"].cron == "0 0 * * * ? *"
         assert results.config["refresh"].time_zone_value == "Etc/UTC"
+        assert results.config["comment"].comment == None
```

Comment:
The test modification is to ensure that when we don't specify a comment, as in this test, we actually read that as None, rather than ''